app/src/main/java/fr/free/nrw/commons/campaigns/CampaignsPresenter.java

```diff
@@ -61,7 +62,7 @@ public void getCampaigns() {
 
                     @Override public void onSuccess(CampaignResponseDTO campaignResponseDTO) {
                         List<Campaign> campaigns = campaignResponseDTO.getCampaigns();
-                        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+                        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd", Locale.ENGLISH);
```

Comment:
Use `dateInLocaleFormat`

Reply:
Can you undo this as IMO we should be fixing the formatting properly instead of putting locale as `English`?